### DISCUSSION TITLE
我是Randomplay店长，今天卡建筑里了 现在谁来救一下

### DISCUSSION THREAD

**Discussion by 56712354** [share121/inter-knot#3193] [其他]
![Screenshot_2024-07-29-14-32-17-74_0dfddd8a234be85a617c2f29d20ccb80](https://github.com/user-attachments/assets/6a38d04b-9a5f-45e2-adf8-504a77076a54)
刚从俱乐部里出来，突然就直接身体进墙里了
谁看到了？去转告一下我哥帮我捞一下。

**Comment by mochengyanliu920** [comment]
这个委托我接了

**Reply by 56712354** [reply]
接了倒是来呀，我要举报你恶意刷单
你未来肯定会被一个传奇绳匠盯上

**Comment by Manbow1997** [comment]
请问，你还在那卡着吗？我这就过去（嘿嘿忍不住笑出声

**Reply by 56712354** [reply]
快找我哥(应该在拉面摊那吃拉面)

**Reply by Manbow1997** [reply]
> 快找我哥(应该在拉面摊那吃拉面)

找你哥多麻烦嘿嘿！你别动啊我马上过来了哈哈哈

**Comment by LilangAlan** [comment]
这个委托我接了！你可以试试菜单的脱离卡死功能<img src="https://bbs-static.miyoushe.com/static/2023/04/28/0c624a8915dd9b24093cf630bded5748_7288657913684710277.png" alt="米仔_拜年" width="75" >

**Reply by 56712354** [reply]
我是新艾利都的，我听不懂。

**Reply by LilangAlan** [reply]
我觉得你得找施工布来把你从建筑里挖出来<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >
